### ISSUE TITLE
Accessibility: multiple use of identical id attributes

### ISSUE THREAD

**Issue by dgruetershopware** [shopware/platform#10305]
### Shopware Version

all

### Affected area / extension

Platform(Default)

### Actual behaviour

Pages often use the same id multiple times - for example:
```
icons-default-arrow-head-left
icons-default-arrow-head-right
```
This violates WCAG success criterion 4.1.1 (Parsing), which requires unique id attributes. Otherwise, this may compromise technical validity and accessibility, especially for screen readers.

### Expected behaviour

The ids should be unique to comply with WCAG success criteria.

### How to reproduce

-

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.

**Comment by mitelg** [comment]
@dgruetershopware do you mean icons in the storefront or administration?

**Comment by dgruetershopware** [comment]
@mitelg Unclear at this point. I've requested more information and will add it as it becomes available.

**Comment by dgruetershopware** [comment]
@mitelg Finally got the info we need;

This is in the storefront. For example, when you have several sliders in a Shopping Experience - the navigator buttons on the slider will all have svg names which are duplicate (or more), like icons-default-arrow-head-left which appears multiple times on that page as a result with the same ID (id=”…” in the browser console). Apparently, they all need to be individually identifiable to be W3C conform.

**Comment by keulinho** [comment]
Closing this as duplicate of #14481, follow the other issue for any updates
